README.md

```diff
@@ -63,6 +69,18 @@ This example targets the C/C++ language, but you can use this for any supported
 3. Add a dependency on `codeql/cpp-all` with `codeql pack add --dir=cpp-customizations codeql/cpp-all`
 4. Implement the customizations module with `mkdir -p cpp-customizations/foo/cpp_customizations && echo "import cpp" > cpp-customizations/foo/cpp_customizations/Customizations.qll`
 
+To verify that the customization pack was correctly imported one can check that it is listed as a dependency in the standard library pack it was added to. It will look as follows (for example for a pack named `foo/cpp-customizations` added to `codeql/cpp-all`):
```

Comment:
Was there a situation where a customization pack was not added?

Perhaps we can add more validation to determine if something looks like a customization pack, but isn't one to warn the user?

Reply:
if the directory naming schema "where any character - in the scope or package_name is replaced with _" 
listed in the readme is not observed , ie if the name accidentally has the `-` not the `_` it will still create the bundle  and it will contain the customization pack but will not do the import into the std lib pack (ie might look sorta correct but will not be usable)

I do know this detail is bold highlighted in the action readme, ... I still have missed it 2/2 times on my own CLI usage attempts 😅 (which is maybe a me thing, but also again, if the user is still only getting familiar with CodeQL overall, making them memorize this specific step is less ideal)

Reply:
tho I would be happy to see a validation step / instead/in addition to the addition to the readme!